docker/Dockerfile

```diff
@@ -0,0 +1,16 @@
+# Docker file for virtual robocup 2021
+
+FROM robocupssl/ubuntu-vnc:latest
+
+# Creates a directory called scripts, copies
+# getRepo script and runs it to fetch the git
+# repository. Once the repository is fetched,
+# it runs setup_software
```

Comment:
this comment just needs to be updated

Reply:
I forgot to save it in vscode before committing! Should be updated now!